src/Microsoft.AspNetCore.Mvc.Core/Resources.resx

```diff
@@ -373,4 +373,7 @@
   <data name="AuthorizeFilter_AuthorizationPolicyCannotBeCreated" xml:space="preserve">
     <value>An {0} cannot be created without a valid instance of {1}.</value>
   </data>
+  <data name="FormCollectionModelBinder_CannotBindToFormCollection" xml:space="preserve">
+    <value>The '{0}' cannot bind to a model of type '{1}'. Change the model type to '{2}' instead.</value>
```

Comment:
Minor but the first "to" and trailing "instead" make this less readable. "cannot bind a model ..." and "change the model type to ..." don't need the extra words.